src/visualizers/widgets/TextEditor/TextEditorWidget.js

```diff
@@ -23,6 +27,18 @@ define([
     MonacoLanguages = JSON.parse(MonacoLanguages);
 
     const AVAILABLE_KEYBINDINGS = ['default', 'vim'];
+    const LANGAUGE_SERVER_HOST = gmeConfig.extensions.languageServers.host || getDefaultServerURL();
```

Comment:
I am not sure that we should have a default hard coded here. I was imagining that, if no language server is specified in the config, it will run without any language servers enabled (rather than assuming it is running on port 5000). Maybe this is what you were referring to when we were talking earlier. If so, my bad - I didn't realize you were referring to hardcoding the port here. 

Since language servers are not required for use, it should proceed without one if none are specified.

Reply:
Agreed will change it. to not use language servers if no host name available.